dspace-api/src/main/java/org/dspace/identifier/doi/DOIConsumer.java

```diff
@@ -18,6 +19,8 @@
 import org.dspace.identifier.IdentifierException;
 import org.dspace.identifier.IdentifierNotFoundException;
 import org.dspace.utils.DSpace;
+import org.dspace.workflow.factory.WorkflowServiceFactory;
+import org.dspace.xmlworkflow.state.Workflow;
```

Comment:
Why is this dependency here, it appears unused (checking in github so not 100% sure)